### PR TITLE
Update MongoDB to 3.4.19, obsolete MongoDB 3.3

### PR DESCRIPTION
**Pull request by Mno-hime**

### CODE REVIEW

File: components/meta-packages/history/history [hunk 1]
```diff
@@ -475,6 +475,8 @@ database/mariadb-55@5.5.55,5.11-2017.0.0.3
 database/mariadb-55/client@5.5.55,5.11-2017.0.0.3
 database/mariadb-55/library@5.5.55,5.11-2017.0.0.3
 database/mariadb-55/tests@5.5.55,5.11-2017.0.0.3
+database/mongodb-common@3.3.12-2018.0.0.3 database/mongodb-common@3.4.19-2018.0.0.0
+database/mongodb-33@3.3.12-2018.0.0.3 database/mongodb-34@3.4.19,5.11-2018.0.0.0
```
Comment (pyhalov) [inline components/meta-packages/history/history]: Perhaps, we should just obsolete it? I'm not sure. In any case, replacing 3.3 with 3.4 on update will give system administrator a lot of work (as well as just removing it).

Comment (Mno-hime) [reply components/meta-packages/history/history]: Right, replacement may bring problems on update. Though, I still think unstable version should not be used, when stable stream is present. How is obsolescence done here? By keeping just the first part? `database/mongodb-common` replacement should be kept as is, I guess?

Comment (pyhalov) [reply components/meta-packages/history/history]: You don't need mongodb-common replacement - it's no-op, package is not versioned.
To mark package obsolete keep just first part.

Comment (Mno-hime) [reply components/meta-packages/history/history]: Obsolete `{service/,}database/mongodb-33`, no replacement now.

**Comment by pyhalov** [comment]
Please, merge two last commits in one.

**Comment by Mno-hime** [comment]
Can you say specifically which commits you have in mind? History of commits is different on Github to what `git log` shows.

**Comment by pyhalov** [comment]
On github web UI I see 3 commits 
"Update MongoDB to 3.4.19"
"Remove unstable MongoDB 3.3 (replaced with stable v3.4)"
"Obsolete MongoDB 3.3"

Can you make it 
"Update MongoDB 3.4 to 3.4.19"
"Obsolete MongoDB 3.3"
?

File: components/meta-packages/history/history [hunk 2]
```diff
@@ -475,6 +475,7 @@ database/mariadb-55@5.5.55,5.11-2017.0.0.3
 database/mariadb-55/client@5.5.55,5.11-2017.0.0.3
 database/mariadb-55/library@5.5.55,5.11-2017.0.0.3
 database/mariadb-55/tests@5.5.55,5.11-2017.0.0.3
+database/mongodb-33@3.3.12-2018.0.0.3
```
Comment (pyhalov) [inline components/meta-packages/history/history]: Current version is 3.3.12-2018.0.0.3. Make this package one version larger (3.3.12,5.11-2018.0.0.4)

File: components/meta-packages/history/history [hunk 3]
```diff
@@ -961,6 +962,7 @@ runtime/tcl-8/snack@2.2.10,5.11-2017.0.0.0
 security/openscap-perl-512@0.8.1,5.11-2014.0.0.0
 security/openscap-python-26@0.8.1,5.11-2014.0.0.0
 security/openscap@0.8.1,5.11-2014.0.0.0
+service/database/mongodb-33@3.3.12-2018.0.0.3
```
Comment (pyhalov) [inline components/meta-packages/history/history]: The same here.

**Pull request merged**